src/dune_threaded_console/dune_threaded_console.ml

```diff
@@ -135,7 +136,7 @@ let progress () =
       (* The current console doesn't react to user events so we just sleep until
           the next loop iteration. Because it doesn't react to user input, it cannot
           modify the UI state, and as a consequence doesn't need the mutex. *)
-      let handle_user_events ~now ~time_budget _ =
+      let handle_user_events ~now ~time_budget _mutex _state =
```

Comment:
Could you add type signatures to ignored values? Just to make sure we notice if the type changes and name becomes stale.